code/core/src/manager/components/sidebar/TagsFilterPanel.tsx

```diff
@@ -177,8 +192,46 @@ export const TagsFilterPanel = ({
 
   const groups = [
     allTags.size === 0 ? [noTags] : [],
-    userEntries.sort((a, b) => a[0].localeCompare(b[0])).map(renderTag),
-    builtInEntries.sort((a, b) => a[0].localeCompare(b[0])).map(renderTag),
+    userEntries.map(renderTag),
+    [
+      renderTag({
+        icon: <DocumentIcon color={color.gold} />,
+        title: 'Documentation',
+        count:
+          (allTags.get('autodocs') || 0) +
+            (allTags.get('attached-mdx') || 0) +
+            (allTags.get('unattached-mdx') || 0) || 0,
+        onToggle: (excluded?: boolean) => {
+          toggleTag('autodocs', excluded);
+          toggleTag('attached-mdx', excluded);
+          toggleTag('unattached-mdx', excluded);
+        },
+        isIncluded:
+          includedTags.has('autodocs') ||
+          includedTags.has('attached-mdx') ||
+          includedTags.has('unattached-mdx'),
+        isExcluded:
+          excludedTags.has('autodocs') ||
+          excludedTags.has('attached-mdx') ||
+          excludedTags.has('unattached-mdx'),
+      }),
+      renderTag({
+        icon: <PlayIcon color={color.seafoam} />,
```

Comment:
Can you use the `PlayHollowIcon` instead? [Looks like this](https://main--64b56e737c0aeefed9d5e675.chromatic.com/?path=/story/icons-playhollowicon--default)